src/openseadragon.js

```diff
@@ -685,19 +685,10 @@
   * @param {OpenSeadragon.Options} options - Viewer options.
   * @returns {OpenSeadragon.Viewer}
   */
-window.OpenSeadragon = window.OpenSeadragon || function( options ){
-
+function OpenSeadragon( options ){
```

Comment:
Good point! I guess we should go with whatever the standard is. Ideally I'd like something that didn't require every plugin to also update their code (though if there is some standard boilerplate OSD plugins should include, we can document that).

I've run across this:

https://github.com/umdjs/umd/blob/master/templates/jqueryPlugin.js

I don't know enough about UMD to know if it sheds any light.

At any rate, just having OSD export its global no matter what would be a reasonable enhancement for plugin support, assuming it's not too much of a violation of UMD practices.